config/manager/manager.yaml

```diff
@@ -55,6 +55,8 @@ spec:
         - "--health-probe-bind-address=:8081"
         - "--metrics-bind-address=127.0.0.1:8080"
         - "--leader-elect"
+        # TODO Need a way to match the container image to the unpack image
+        - "--unpack-image=quay.io/operator-framework/operator-controller:devel"
```

Comment:
How about:
```suggestion
        - "--unpack-image=quay.io/operator-framework/operator-controller-unpack:devel"
```

Reply:
Are you suggesting this change in order to more explicitly indicate that this string is being replaced, or in order to use a separate image? 
If the former, the latest commit should take care of that, but if the latter, I think it's better to have the `unpack` binary in the same image as the `manager` so that we don't have to build, push, and manage two separate images. Rukpak also used this strategy FWIW.

Reply:
Please ignore - Just realised from the Dockerfile that operator controller and unpack image are going to be the same.